EventListener/FilterEventListener.php

```diff
@@ -94,7 +94,6 @@ private function addQueryBuilderFilter()
 
                 $filter = $filterExtension->getFilters()[$column];
 
-                // TODO: automatically join (split field on '.')
```

Comment:
@rugbymauri This should not be removed, seeing as it was not implemented yet. A `TODO` is not necessarily a bad thing, but should serve as a pointer for possible future enhancements.